### PR TITLE
[tests-only] Added spaces examples in apiWebdavLocks2 suite

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Related Issue

- Part of https://github.com/owncloud/ocis/issues/3098

## How Has This Been Tested?

- :robot:
- https://github.com/owncloud/ocis/pull/3125

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiTranslation-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34558/82/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareReshareToShares2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34558/76/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>cliAppManagement-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34558/119/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFederationToRoot2-latest-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34558/106/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareReshareToRoot3-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34558/77/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavLocks-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34558/88/1

File: tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature [hunk 1]
```diff
@@ -103,6 +117,12 @@ Feature: lock should propagate correctly if a share is reshared
       | new      | shared     |
       | new      | exclusive  |
 
+    @personalSpace @skipOnOcV10
+    Examples:
+      | dav-path | lock-scope |
+      | spaces   | shared     |
+      | spaces   | exclusive  |
+
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature:125]:
```suggestion


```

**Review by amrita-shrestha — COMMENTED** [review]
others look LGTM

File: tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature:125]: thank god it was the last scenario on the file :grin: 
thx! fixed!

**Review by phil-davis — APPROVED** [review]

**Pull request merged**